modules/nw-ingress-sharding.adoc

```diff
@@ -1,17 +1,105 @@
+
+When Ingress Controllers are sharded, a given route is admitted to zero or more Ingress Controllers in the group. A route's status describes whether an Ingress Controller has admitted it or not. An Ingress Controller will only admit a route if it is unique to its shard.
+
+An Ingress Controller can use three sharding methods:
+
+. Adding only a namespace selector to the Ingress Controller, so that all routes in a namespace with labels that match the namespace selector are in the Ingress shard.
+
+. Adding only a route selector to the Ingress Controller, so that all routes with labels that match the route selector are in the Ingress shard.
+
+. Adding both a namespace selector and route selector to the Ingress Controller, so that routes with labels that match the route selector in a namespace with labels that match the namespace selector are in the Ingress shard.
+
+This lets you distribute subsets of routes over multiple Ingress Controllers. These subsets can be non-overlapping, also called _traditional_ sharding, or overlapping, otherwise known as _overlapped_ sharding.
+
+== Traditional sharding example
+
+An Ingress Controller `finops-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `finance` and `ops`:
+
+.Example YAML definition for `finops-router`
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: finops-router
+    namespace: openshift-ingress-operator
+  spec:
+    namespaceSelector:
+      matchLabels:
+        name:
+          - finance
+          - ops
+----
+
+A second Ingress Controller `dev-router` is configured with the label selector `spec.namespaceSelector.matchLabels.name` set to `dev`:
+
+.Example YAML definition for `dev-router`
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: operator.openshift.io/v1
+  kind: IngressController
+  metadata:
+    name: dev-router
+    namespace: openshift-ingress-operator
+  spec:
+    namespaceSelector:
+      matchLabels:
+        name: dev
+----
+
+If all application routes are in separate namespaces, each labeled with `name:finance`, `name:ops`, and `name:dev` respectively, this configuration effectively distributes your routes between the two Ingress Controllers. {product-title} routes for console, authentication, and other purposes should not be handled.
+
+In the above scenario, sharding becomes a special case of partitioning, with no overlapping subsets. Routes are divided between router shards.
+
+[WARNING]
+====
+The `default` Ingress Controller will continue to serve all routes unless the `namespaceSelector` or `routeSelector` fields contain routes that are meant for exclusion. See this link:https://access.redhat.com/solutions/5097511[Red Hat Knowledgebase solution] and the section "Sharding the default Ingress Controller" for more information on how to exclude routes from the default Ingress Controller.
```

Comment:
Does this work just as well?: "will continue to serve" -> "serves"

Reply:
maybe, but it's a good point, I'll update it. I'm trying to convey that even if you create custom Ingress Controllers to handle sharding, it doesn't change the behavior of the default Ingress Controller...the custom ones use kind of an inclusive logic in their config and the default uses more of an exclusive logic. So maybe "continues to serve" or something

Reply:
Could just be explicit about it in the admonition, unless that information is definitely going to be in the reader's head at this point. 

It's admittedly different doing a peer review in a code editor than it is following docs as a real user, so I could be off base. :+1: